bundles/org.openhab.binding.buienradar/src/main/java/org/openhab/binding/buienradar/internal/BuienradarHandler.java

```diff
@@ -54,15 +57,15 @@
 
     private final Logger logger = LoggerFactory.getLogger(BuienradarHandler.class);
 
+    private static final Unit<Speed> MILLIMETRE_PER_HOUR = Units.METRE.divide(1000).divide(Units.HOUR)
```

Comment:
Why don't we add that to the smarthome units?

Reply:
It is already added. However, I cannot test this binding on my RPi because it is only in 2.5.0-SNAPSHOT.

Reply:
That would make testing a lot harder for me (and other contributors). I mean, essentially we are programming the bindings against the 2.4.0 API, right? There is no forced upgrade path, the API is backward compatible. Otherwise we'd have an ordering problem in the commits as well.

For now (I don't want to argue endlessly), I'll just fix this opaque, almost undocumented call to the `Units` API with `MILLIMETRE_PER_HOUR` from the arbitrary mishmash of UoM in `SmartHomeUnits`

Btw, where is the documentation of the uom library? The [github](https://github.com/unitsofmeasurement/uom-se) page has a minimal `README.md`, JSR 363 I have no idea where to begin (don't click on the first link in the list below, `http://www.unitsofmeasurement.org/` will redirect to a spam domain), even Martin Fowlers site on quantities is no longer online (https://martinfowler.com/eaaDev/quantity.html) and Googling I cannot find a good list of examples (except in Kotlin...)

Don't get me wrong, I am *all* *for* type-safety when dealing with quantities. It's just that I am spoiled with Scala's Squants.